server/task_executor_pitr_cutover.go

```diff
@@ -95,20 +96,18 @@ func (exec *PITRCutoverTaskExecutor) pitrCutover(ctx context.Context, task *api.
 	}
 	defer conn.Close()
 
-	mysqlDriver, ok := driver.(*mysql.Driver)
-	if !ok {
-		log.Error("failed to cast driver to mysql.Driver")
-		return true, nil, fmt.Errorf("[internal] cast driver to mysql.Driver failed")
-	}
-	mysqlDriver.SetUpForPITR(exec.mysqlutil, binlogDir)
-
-	log.Debug("Swapping the original and PITR database.", zap.String("instance", task.Instance.Name), zap.String("originalDatabase", task.Database.Name))
+	log.Debug("Swapping the original and PITR database", zap.String("originalDatabase", task.Database.Name))
 	pitrDatabaseName, pitrOldDatabaseName, err := mysql.SwapPITRDatabase(ctx, conn, task.Database.Name, issue.CreatedTs)
 	if err != nil {
-		log.Error("Failed to swap databases and backup the original database", zap.Error(err))
+		log.Error("Failed to swap the original and PITR database", zap.String("originalDatabase", task.Database.Name), zap.String("pitrDatabase", pitrDatabaseName), zap.Error(err))
 		return true, nil, fmt.Errorf("failed to swap the original and PITR database, error: %w", err)
 	}
-	log.Debug("Finish swapping the original and PITR database", zap.String("original_database", task.Database.Name), zap.String("pitr_database", pitrDatabaseName), zap.String("old_database", pitrOldDatabaseName))
+	log.Debug("Finished swapping the original and PITR database", zap.String("originalDatabase", task.Database.Name), zap.String("pitrDatabase", pitrDatabaseName), zap.String("oldDatabase", pitrOldDatabaseName))
+
+	backupName := fmt.Sprintf("%s-%s-pitr_%d", api.ProjectShortSlug(task.Database.Project), api.EnvSlug(task.Database.Instance.Environment), issue.CreatedTs)
```

Comment:
Use dash instead of underscore for consistency?

Reply:
SG.